PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -100,12 +100,19 @@ UpdateLoadedImage (
       return EFI_UNSUPPORTED;
     }
 
+    //
+    // TODO: handle the rest of the container components
+    //
```

Comment:
remove the TODO.

https://slimbootloader.github.io/tools/GenContainer.html#normal says it can have only one image

Reply:
Done. Thanks!